UPDATING.md

```diff
@@ -130,6 +130,62 @@ session.add(admin)
 session.commit()
 ```
 
+
```

Comment:
could you put this in L27 (the first item under Airflow master section)? I believe the file is organized in reverse chronological order

Reply:
Sure. Fixed.